lmdeploy/cli/chat.py

```diff
@@ -65,7 +65,7 @@ def add_parser_turbomind():
         ArgumentHelper.session_len(engine_group)
         # other arguments
         ArgumentHelper.cap(parser)
-        ArgumentHelper.meta_instruction(parser)
+        ArgumentHelper.chat_template(parser)
```

Comment:
What's the motivation? It will cause CLI bc breaking

Reply:
Let's keep this PR focusing on adjusting `request_output_len` and return `history_tokens`

Please open another PR for `chat_template`